docs/sources/mimir/operators-guide/run-production-environment/scaling-out.md

```diff
@@ -20,6 +20,16 @@ However, be careful when scaling down some of the stateful components as these a
 When running Grafana Mimir in monolithic mode, you can safely scale up to any number of instances.
 To scale down the Grafana Mimir cluster, see [Scaling down ingesters](#scaling-down-ingesters).
 
+## Read-Write mode
+
+When running Grafana Mimir in read-write mode, you can safely scale up any of the three components.
+
+The Mimir read component can be safely scaled up and down as it is stateless (an autoscaler may also be used).
+
+The Mimir backend component can be safely scaled down in one zone at a time. As it contains the store-gateway, see [Scaling down store-gateways](#scaling-down-store-gateways).
+
+To scale down the Mimir write component, see [Scaling down ingesters](#scaling-down-ingesters).
```

Comment:
```suggestion
* You can safely scale the Mimir read component up or down because it is stateless. You could also use an autoscaler.

* You can safely scale the Mimir backend component down within one zone at a time. Because it contains the store-gateway, for more information see [Scaling down store-gateways](#scaling-down-store-gateways).

* To scale down the Mimir write component, see [Scaling down ingesters](#scaling-down-ingesters).
```

Reply:
Please vet this  "as" -> "because" change for accuracy. (I assume you did not mean "while".)

Reply:
Yep, thank you.